### DISCUSSION TITLE
Morphometric doesn't work

### DISCUSSION THREAD

**Discussion by Empanag** [UMEP-dev/UMEP#480] [General]
I m trying to use morphometric tool for grid using only buildings file and a shapefile but i don't have any outputs only the isotropic file which is empty.

my data are these in the link
["D:\FORTH-RSLab Dropbox\FORTH-RSLab Dropbox\RSLab\Manolis Panagiotakis\MorphometricInputs.zip"](https://www.dropbox.com/s/zii3volqzc4k7dg/MorphometricInputs.zip?dl=0)

**Comment by biglimp** [comment]
You cannot have Nodata pixels representing the ground in your DSM. Replace them with zeros and try again. I recommend to use **Fill NoData Cells** in the Processing toolbox in QGIS.
